R/fit_model.R

```diff
@@ -75,17 +62,24 @@ fit_model <- function(
         rt = rt,
         gp = gp,
         stan = stan,
-        verbose = interactive()
+        verbose = TRUE,
+        # Dump logs to console to be caught by pipeline's logging instead of
+        # EpiNow2's default through futile.logger
+        logs = EpiNow2::setup_logging(
+          threshold = "INFO",
+          file = NULL,
+          mirror_to_console = TRUE,
+          name = "EpiNow2"
+        ),
```

Comment:
@seabbs do you see something weird with this specification? I tried to follow the docs exactly but this is launching a browser window with the path as the EpiNow2 logs when I run the pipeline. Not a total dealbreaker but a weird thing I'd like to fix.

Reply:
> launching a browser window with the path as the EpiNow2 logs when I run the pipeline.

Wait what?

Reply:
Now https://github.com/CDCgov/cfa-epinow2-pipeline/issues/101